packages/react/package.json

```diff
@@ -54,7 +54,7 @@
     "test:e2e:update": "cypress-image-diff -u",
     "test:e2e": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress run-ct --browser chrome --config video=false,screenshotOnRunFailure=false",
     "test:e2e:interactive": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress open-ct --browser chrome --config video=false,screenshotOnRunFailure=false",
-    "test:e2e:ci": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress run-ct  --browser chrome --headless --config --record video=false,screenshotOnRunFailure=false",
+    "test:e2e:ci": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress run-ct  --browser chrome --headless --config watchForFileChanges=false,retries=3 --record video=false,screenshotOnRunFailure=false",
```

Comment:
Adding retries as it's the recommended practice in cypress since many aspects as considered flakey. This seems to fix the intermittent failures. https://docs.cypress.io/guides/guides/test-retries#Introduction